robot/control/main.cpp

```diff
@@ -145,51 +195,12 @@ int main() {
     moduleList.emplace_back(curTime, KickerModule::period,        KickerModule::runtime,        &kicker);
     moduleList.emplace_back(curTime, BatteryModule::period,       BatteryModule::runtime,       &battery);
     moduleList.emplace_back(curTime, RotaryDialModule::period,    RotaryDialModule::runtime,    &dial);
-    moduleList.emplace_back(curTime, LEDModule::period,           LEDModule::runtime,           &led);
+    moduleList.emplace_back(curTime, LEDModule::period,           LEDModule::runtime,           led.get());
 
-    while (true) {
-        // Must do all timing in systick
-        // Base time we use is in systick, and if you convert from systick
-        // to us and try to compare timepoints, the value will "overflow" at
-        // some random point which messes up the `Current - Start` magic
-        // that automatically deals with delta time using the math behind overflows
-        uint32_t loopStartTime = DWT_GetTick();
-        uint32_t loopEndTime = DWT_GetTick() + SUPER_LOOP_PERIOD*DWT_SysTick_To_us();
+    osThreadDef(SuperLoopTask, superLoopTask, osPriorityAboveNormal, 1, 1 << 15);
+    osThreadCreate(osThread(SuperLoopTask), NULL);
```

Comment:
I assume we don't really care if threads fail to init. The entire system will be dying anyways

Reply:
Yeah, this whole thing will be getting the axe. This is just to get RTOS stuff up-and-running ASAP so we can verify things don't break.

Reply:
Yeah, I'm okay with not handling errors for now. This will be getting the axe soon, it's just to make sure RTOS doesn't break anything horrifically and to keep the process moving.